templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,293 @@
 # Import from Slack
 
-Starting with Zulip 1.8, Zulip supports importing data from Slack,
-including users, channels, messages, attachments, avatars, custom
-emoji, and emoji reactions.
+You can import your Slack organization into Zulip. It's a great way to preserve
+your organization's history when you migrate from Slack to Zulip, and to make
+the transition easy for the members of your organization.
 
-This tool has been used to import Slack workspaces with 10,000 members
-and millions of messages. If you're planning on doing an import much
-larger than that, or run into performance issues when importing,
-[contact us](/help/contact-support) for help.
+The import will include:
 
-**Note:** You can only import a Slack workspace as a new Zulip
-organization. In particular, you cannot use this tool to import from Slack
-into an existing Zulip organization.
```

Comment:
Hmm, perhaps we should preserve this note in some way? It might be important for users' understanding this "adding" their Slack data to their pre-existing org is not possible

Reply:
I tried changing the warning to:

>     You can only import a Slack workspace as a new Zulip organization. Slack
    workspace history cannot be added into an existing Zulip organization.